modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -1,21 +1,99 @@
-reliability, and security requirements. These clusters are also deployed with an
-integrated and dedicated Redpanda Console and, optionally, with a fully managed
-Kafka Connect and connectors cluster.
+Redpanda Cloud is a complete streaming data platform delivered as a full-managed service, with automated upgrades and patching, data and partition balancing, and 24x7 support. It continuously monitors and maintains your clusters along with the underlying infrastructure to meet strict performance, availability, reliability, and security requirements. All Redpanda Cloud clusters are deployed with an integrated Redpanda Console.
+
+== Cluster types
+
+Redpanda offers three types of fully-managed cloud clusters:
+
+* <<Serverless>>: Multi-tenant clusters are hosted in Redpanda Cloud. This is the fastest and easiest way to start data streaming in the cloud. With pay-as-you-go billing, you only pay for what you consume.
+* <<Dedicated Cloud>>: Single-tenant clusters are hosted in Redpanda Cloud. This provides more control over your deployment. 
+* <<Bring Your Own Cloud (BYOC)>>: Clusters are hosted in your private cloud. This provides more control over your deployment and offers full data sovereignty. You can subscribe with committed use in Google Cloud Marketplace or AWS Marketplace. 
+
+=== Serverless
+
+With Serverless clusters, you host your data in Redpanda's VPC, and Redpanda handles provisioning, operations, and maintenance. Serverless clusters are multi-tenant deployments, which means that your data is hosted on Redpanda clusters shared among multiple tenants. Other tenants never access your data, and your data never leaves your selected region. 
+
+Your applications talk to Serverless over the public internet using encrypted connections. Serverless does not support private networking (that is, VPC peering to talk over private IPs). Use Dedicated clusters for private networking.
+
+Available on AWS.
+
+=== Dedicated Cloud
+
+With Dedicated clusters, you host your data in Redpanda's VPC, and Redpanda handles provisioning, operations, and maintenance. Dedicated clusters are single-tenant deployments for better data isolation. 
+
+Available on AWS or GCP. 
+
+=== Bring Your Own Cloud (BYOC)
+
+With BYOC clusters, you deploy Redpanda in your own VPC, so all data is
+contained in your own environment. There are no inbound connections from the Redpanda
+control plane to the data plane. This provides an additional layer of security
+and isolation. Redpanda handles provisioning, operations, and maintenance. 
+
+Setting up a BYOC cluster is initiated from the Redpanda control plane. This is a two-step process performed using `rpk`:
+
+. You bootstrap a virtual machine (VM) in your VPC. This VM spins up the agent
+and the required infrastructure.
+. The agent communicates with the control plane to pull the cluster specifications.
+
+When provisioning BYOC, at the time an agent is deployed, Redpanda assigns the
```

Comment:
I might reword this for clarity.

"Redpanda assigns the necessary IAM policies required to run the agent at the time of deployment."